### DISCUSSION TITLE
Forcing "hidden" bosses with ff2_override denies action. How to give access to admins to force hidden bosses?

### DISCUSSION THREAD

**Discussion by naydef** [Batfoxkid/Freak-Fortress-2-Rewrite#145] [Q&A] [answered]
As the title says, using `ff2_override` on hidden boss prints [`FF2] You don't have permission to select [Hidden Boss] Some boss`.
One needs to execute command from console to force a hidden boss. So the question is, how to give myself perms to force any boss (hidden or not).

**Comment by Batfoxkid** [comment, marked as answer]
Admins need the same permissions to access said boss in `ff2_override`
To enable all bosses to appear for said admin, use the command override `ff2_all_bosses` and set it to the appropriate admin flag

**Reply by naydef** [reply]
Hm, ok. I didn't try to set `ff2_override` yet, but on the server I have root permissions and I still doesn't allow me to force hidden boss. Usually commands should unconditionally give access to root players.
